### ISSUE TITLE
Vue.js Community Translations

### ISSUE THREAD

**Issue by bencodezen** [vuejs/docs-next#478]
This serves as a list of announcements for community translations. Instead of creating a new issue, a new translation announcement (along with its description, repository URL, call for contributors etc.) should be posted here as a comment. Further discussions and progress tracking for a specific translation should happen in the corresponding repository.

**Comment by ErickPetru** [comment]
Hi @bencodezen, so we're starting our **Brazilian Portuguese** (_pt-BR_) translation work at the fork: https://github.com/vuejs-br/docs-next. Any Portuguese speaker whiling to help us please go there and create or join on an _issue_ about the related `.md` file.

**Comment by Alex-Sokolov** [comment]
**Russian** translation — https://github.com/translation-gang/docs-next

**Comment by NatanCieplinski** [comment]
Hi, i'm currently working on the Italian translation of Vue 2. Can the v3 doc be considered stable enough to abandon the translation of the v2 and start the translation of this one?
If not, is there any date by which i can consider the doc ready for translation?

**Comment by animyrch** [comment]
Hello, first of all, congratulations to the Docs team on your work.

I was the maintainer of the Turkish translation of the Vue.js 2.0. I will be continuing the translations of the Vue3 doc on this repository : https://github.com/animyrch/docs-next-tr

As a feedback, I would like note that the lack of integration of the Turkish translation link within the language selector was demotivating and as a result, I was never able to finish the translations. Hoping that the experience as a translation contributor will be better this time around.

**Comment by narusas** [comment]
Hi.
Vue Korean User group working on Korean translation for vue3.
Fork Repo: https://github.com/narusas/docs-next

Translation is on going. 

Anyone want join us, contact vuejs-korea.slack.com slack.

**Comment by siu91** [comment]
Chinese translation of Vue.js document is in progress...
Fork repo: https://github.com/vuejs-docs/docs-next-zh，and join us.

**Comment by siu91** [comment]
Chinese translation of Vue.js document is in progress...
Fork repo: https://github.com/vuejs-docs/docs-next-zh

**Comment by farzadso** [comment]
Hi.

The Persian (Farsi) translations will be carried out in this fork : https://github.com/vuejs-fa/docs-next

If you want to give a helping hand, you can message me on [Twitter](https://twitter.com/farzadso).

**Comment by lagp0310** [comment]
Hi @bencodezen! I'm starting the translation to Spanish in this repository: https://github.com/lagp0310/docs-next. Contributions are welcome!

**Comment by NatanCieplinski** [comment]
Hi,
i started the Italian translation at https://github.com/NatanCieplinski/docs-next-it
If anyone wants to join, just contact me.

**Comment by animyrch** [comment]
In the previous Docs, many people posted a checklist for articles to be translated (makes sense for project tracking ) but it was cluttering the thread. I think this is now discouraged. In any case, it is much better to do this sort of tracking through an issues list on your own repo. 
Given the high number of files (and therefore potential issues), I created an npm module that takes care of this for you. It takes a list of issue definitions as a json file and creates the issues on your GitHub repository automatically.
https://www.npmjs.com/package/issuecreator

For even more ease of use, you can find the already-created json file below as a zip file that can be used for all of the files in docs-next project.

[issues.zip](https://github.com/vuejs/docs-next/files/5279034/issues.zip)

**Comment by lagp0310** [comment]
Hi everyone!

I think it could be useful to keep track of which files in a repository are translated and which are not, yet. That's why I made a small tool that can analyze the language of files in a repository, output the result and build a Markdown table.

For an example of the results, please check the [Vue Docs Next Spanish](https://github.com/lagp0310/docs-next#progreso-de-la-traducci%C3%B3n) repository.

The tool is [here](https://github.com/lagp0310/markdown-docs-translation-tracker).

**Comment by mandaputtra** [comment]
Hello everyone me and my friends on [@vuejs-id](https://vuejs.id/) will be translating the docs onto Indonesian. The repo is [here](https://github.com/vuejs-id/docs-next). We make use of [wei/pull](https://github.com/wei/pull) bot to keep track on the `master` branch so it automatically makes a pull request. Although there maybe a conflict in the future if the translation are final, but its still better than manually apply update per commit.

**Comment by yuesong-feng** [comment]
**简体中文**
Hi everyone! 
As a big fan of Vue, I am about to work on the **Chinese** translation of Vue 3.0 document. You can find on the following link: 
[https://github.com/yuesong-feng/docs-next](url)
If you are interested in the translation or if you find errors of my translation, please fell free to drop me an email.

**Comment by narusas** [comment]
@bencodezen Hi
Korean translation is almost done. 
If we finished, How to submit ? or Some kind of Rule for Translation PR?

**Comment by narusas** [comment]
vue  3 korean translation is here!!  https://v3.vuejs-korea.org/ko-KR/ 

@bencodezen Can I make PR to origin?

**Comment by narusas** [comment]
#756 

Hello, I just finished translate in Korean.

https://github.com/narusas/docs-next

and We run it at

https://v3.vuejs-korea.org/ko-KR/

Please add v3.vuejs.org :)

**Comment by veaba** [comment]
Chinese translation for v3:
- repo: https://github.com/vuejs/docs-next-zh-cn
- website: https://v3.cn.vuejs.org
- status: is ready.

**Comment by AlexRMU** [comment]
@bencodezen,
how about https://crowdin.com/?

**Comment by AlexRMU** [comment]
> how about https://crowdin.com/?

It is a bit paid, but maybe there are analogues?
So that everything is centralized and organized

**Comment by demahom18** [comment]
### French translation 
Repo:  https://github.com/demahom18/docs-next

**Comment by OtabekSadiridinov** [comment]
Today, I forked docs-next. And for the first time I am going to try to translate VueJS docs into Uzbek language.
---
Bugun, Men doc-next repositoriyasini fork qildim. Va birninchi marta VueJS qo'llanmasini O'zbek tiliga tarjima qilishni boshlayapman.

**Comment by Gismo1337** [comment]
### GERMAN
**in Progress**
Vue 3:

- [ x ] Essentials

I am a beginner at git, github and vue.  I started the translation to learn. 
please be kind if i made mistakes.

Repo: [https://github.com/Gismo1337/docs-next-ger](https://github.com/Gismo1337/docs-next-ger)

**Comment by eminkokdemir** [comment]
Greetings.
I see that a project has been started as a Turkish translation, but it is not finished. I started translation for my personal development and for the benefit of our Turkish-speaking friends. If you want to contribute, I expect a repo.
https://github.com/eminkokdemir/tr.vuejs.org

**Comment by Gismo1337** [comment]
Hey there. i do the translation to german, not turkish language.

Am Fr., 28. Mai 2021 um 17:05 Uhr schrieb eminkokdemir <
***@***.***>:

**Comment by Oleksii14** [comment]
Ukrainian is coming!

in Progress

Repo: https://github.com/Oleksii14/vuejs-uk-translation

**Comment by dogia** [comment]
How can I help to translate English to Spanish?

**Comment by milindanath** [comment]
Hi,
I have started translating VueJs docs to Sinhala language today. 

Repo: https://github.com/milindanath/docs-lk-si
Deployed to: https://v3-si-docs.netlify.app/

Please feel free to join if anyone interested.

**Comment by kei-ichi** [comment]
Hi. @bencodezen 
It look like Vue.js v2 Vietnamese docs is not complete at all but I'm working on Vietnamese translation for Vue.js v3.

Repo: [https://github.com/vuejs-vi/vue-docs](https://github.com/vuejs-vi/vue-docs)
Deploy to: [https://vuejs-vi.org](https://vuejs-vi.org)

Anyone want to help me translate the docs, feel free to contact me at [hue@kei-ichi.com](hue@kei-ichi.com)

**Comment by giacomomaccanti** [comment]
> Hi,
> i started the Italian translation at https://github.com/NatanCieplinski/docs-next-it
> If anyone wants to join, just contact me.

Hello,
I thought I was participating; do we get in touch?

**Comment by bencodezen** [comment]
Hey @kei-ichi! I'm going to tag @phanan who has more context into the Vietnamese translation community. Thanks for your efforts on this!

**Comment by kei-ichi** [comment]
> Hey @kei-ichi! I'm going to tag @phanan who has more context into the Vietnamese translation community. Thanks for your efforts on this!
-----------
Thank you @bencodezen for your response. The reason I (and my some friends) decided to create another Github organization for Vietnamese Vue community are:
- The repository for Vue v2 doc has been archived even it isn't complete translated to Vietnamese
- Complete zero translation about: Vue v2 API docs, Vue v2 cookbook, Vue router, Vuex, and of course other popular Vue v2 plugin like Vue-vue-i18n
- The last commit from his (phanan) Vue 3 docs is 10 month ago (Jan 4 2021).

I will edit my previous post, and suspense my translate team till June 2022 (6 months from now). If he and his translate team still do anything, I will contact you for continuous my current work. Are you ok with that?

**Comment by phanan** [comment]
@kei-ichi There's indeed a pause (or in this case I guess, a stop) to v2 translation now that v3 is out and automatically has the main focus. We maintain an official [vuejs-vn](https://github.com/vuejs-vn) organization, which hosts the official translation repositories, so the best way to proceed IMO would be to have you as part of the org and go from there. Before that, would you mind sharing with me the current state of your translation? Both of your provided links are 404 currently.

Also, in order to keep this thread focused, feel free to shoot an email at me@phanan.net.

**Comment by maogongzi** [comment]
¡Hola a todos mis amigos hispanohablantes, espero que os encontráis muy muy bien! Estoy trabajando en la traducción de [https://vue3-docs-es.netlify.app/](https://vue3-docs-es.netlify.app/) aquí en este repo: [https://github.com/maogongzi/docs/](https://github.com/maogongzi/docs/), ahora he terminado con las secciones de **Essentials**, mi plan siguiente es trabajar con **Components In-Depth**, depende del tiempo libre que yo tengo.

Me he dado cuenta de que @lagp0310 estaba traduciendolo pero no se porque el repo está archivado ahora, por eso decidí empezar con un nuevo repo, pero basado a los archivos ya completados por @lagp0310 lo más posible, para otros, primero busco por el proyecto de la documentación de Vue 2 para las traducciones y términos ya existentes, si no los encuentro trato de traducirlos desde cero.

Yo mi mismo  no es un hablante nativo, es el amor de Español que me impulsa a hacerlo, por eso si se encuentran errores en los documentos traducidos, es un placer para mí que alguien abra un issue o envíe un PR para arreglarlos :) Si hay amigos también están interesado en esto, podríamos trabajar juntos para llevar la documentación de Vue 3 a Español, gracias!

**Comment by carlosepcc** [comment]
> mentación de Vue 3 a Español, gracias!

Genial! Es lo que estaba buscando. Soy hablante nativo pero será mi primera vez colaborando en Github en general. Espero que este repo llegue a publicarse en la pagina oficial.

¿Se puede usar en este repo la herramienta [lagp0310/markdown-docs-translation-tracker](https://github.com/lagp0310/markdown-docs-translation-tracker)?

**Comment by maogongzi** [comment]
@carlosepcc ¡por supuesto! Déjame investigar cómo incorporarlo en mi repo, por ahora, puedes explorar la lista de issues cerrados para ver los archivos ya completados, y si encuentras un error(quizá muchos...), no dudes decírmelo! (o mejor enviar un PR, :D)

**Comment by anchan42** [comment]
Thai translation started as part of my learning process. :) 
https://github.com/anchan42/Vue-3-Thai-docs

- [x]  Essentials
- [ ]  Components In-Depth
- [ ]  Transitions & Animation
- [ ]  Reusability & Composition
- [ ]  Advanced Guides
- [ ]  Tooling
- [ ]  Scaling Up
- [ ]  Accessibility

**Comment by dev-oiram** [comment]
@maogongzi for Spanish translation how are you working since I don´t see any PR created on the repo https://github.com/maogongzi/docs
Also how do you manage the issues?
I want to help with the translation but are you guys working in a certain way?

**Comment by omiras** [comment]
@dev-oiram @maogongzi @lagp0310  Same here. I would like to help but never really been involved in the past with projects translations. How should we start?

¿Cómo podemos ponernos de acuerdo sobre en qué repositorio continuar el trabajo de traducción e incorporarlo a Vue?

**Comment by dev-oiram** [comment]
@omiras  @maogongzi  I can start by checking for mistakes on the files that have already been translated and of course start the translation with some new ones. I´m gona make pull request to see if someone @omiras  @maogongzi  can make a review before the merge to main branch by @maogongzi . If you have any questions i can help you and we can make reviews before any merge.

**Comment by maogongzi** [comment]
@dev-oiram @omiras hello! sorry for replying late as I was on the Chinese New Year vacation, thank you friends for joining, definitely good news to push the translation progress forward, speaking of how do I manage the repo, currently I create one issue per file to be translated, and link one corresponding PR to translate/optimize it, there are already closed PRs out there in the repo you can take a look :)

Fork and PR to collaborate sounds good to me, but I can also assign access of my repo to you so you can push directly to my repo like your own, so that things will get even smoother.

Also one thing need to be noticed is that the official docs team is doing a giant refactoring on the whole docs project of v3, so maybe we should halt the new translations but focus mainly on correcting the existing ones, so that later it'll be easier to synchronize with the new official docs, what's you opinion? @dev-oiram @omiras

**Comment by dev-oiram** [comment]
@maogongzi  its ok if i start checking the files for any error or you want me to start translate the new ones. I´m going to start in a week from the date of this post since I´m finishing a project

I want to help as much as I can.

**Comment by maogongzi** [comment]
@dev-oiram @omiras  I have assigned access for you to my repo and ¡bienvenidos a bordo!

p.s. you can start by checking the existed files, or just start the new ones in the "migration from v2" section, either one is good to me! for other sections, I'm afraid there will be big changes due to the refactoring, so maybe we handle them later?

my usual working flow:

1. create an issue for one file (untranslated, with its path in issue title)
2. create a `feature/blabla` branch to work on the issue
3. create a PR using the feature branch pointing to `master` (not `main` for now)
4. link the PR to the issue, so later I know what PR is for what purpose
5. once a PR is merged into `master`, the netlify deploying process will be triggered automatically and several minutes later visit the preview link to see the results.
6. after I finish some files I'll run the script provided by @lagp0310 (the script is tweaked a bit by me to better fit the repo) to do a statics on which files have been translated and commit to `master` branch, so I can see the green ✅ s out there.

**Comment by ManuSquall** [comment]
Hello world, 

About the french translation of the doc, @demahom18 , @edimitchel, in which one of your repo should we contribute, you both have begin a fork translation [here](https://github.com/edimitchel/docs-fr) and [here](https://github.com/demahom18/docs-next)??

**Comment by edimitchel** [comment]
> Hello world, 
> 
> About the french translation of the doc, @demahom18 , @edimitchel, in which one of your repo should we contribute, you both have begin a fork translation [here](https://github.com/edimitchel/docs-fr) and [here](https://github.com/demahom18/docs-next)??

We focus all efforts on my repo as the @demahom18 was based on a old version and was done with only one person.
To avoid any mistake, doing translation with many people is better.
So let's work together, @demahom18 contributor on it too :)

**Comment by matrunchyk** [comment]
### 🇺🇦 Ukrainian translation 🇺🇦:

https://github.com/vuejsorgua/docs-next

Translation is available [here](https://vuejs.org.ua/).

It's still WIP.

**Comment by veaba** [comment]
你好先生/女士！     当你看到本回复，说明你的邮件我已经收到，我会尽快回复处理你的邮件，祝你愉快。      ***@***.***

**Comment by thetejasmagade** [comment]
Hi developers! I'm starting the translation to "Hindi" in this repository: https://github.com/thetejasmagade/hn.vue.docs Contributions are welcome! 

Thank you

**Comment by miguelfdezc** [comment]
Hi! @maogongzi I would like to start contributing to the "Spanish" translation of the documentation too 🙂

Can I start by making a PR of any of the existing issues? Or should I ask you to assign me one first

**Comment by angelhdzmultimedia** [comment]
Any update to include the spanish (español) translations? https://vue3-spanish.netlify.app/guide/reusability/plugins.html

**Comment by Ibochkarev** [comment]
**Russian translation**

GitHub: https://github.com/translation-gang/docs-ru

Website (Temporarily until moving to the official domain): https://docs-ru.vercel.app/

Telegram: https://t.me/vuejs_docs_ru
